tests/test_file_generation.py

```diff
@@ -194,24 +194,11 @@ def test_different_roots_graph() -> None:
 
 
 @pytest.mark.parametrize(
-    "file_name, expected_top_level_keys, expected_file_path_level_1, "
-    "expected_file_path_level_2, expected_breakpoints",
+    "file_name, expected_top_level_keys, expected_breakpoints",
```

Comment:
instead of removing parts of the test, you could make it non-parameterized. extract a helper function `file_name` to `opossum_information` and then have two similar tests.

Reply:
Yes, good idea! I changed the tests accordingly.